Source/WebCore/rendering/BreakLines.h

```diff
@@ -59,6 +59,12 @@ class BreakLines {
     template<NoBreakSpaceBehavior nonBreakingSpaceBehavior>
     static inline bool isBreakableSpace(UChar character);
 
+    static inline bool isASCIILetter(UChar character)
+    {
+        return ('a' <= character && character <= 'z') // Check lowercase first.
+            || ('A' <= character && character <= 'Z');
+    }
```

Comment:
You can use `isASCIIAlpha` in WTF, which further reduces branches via bit hack.

Reply:
Done, thanks!